File: baekjoon/step/10250.py
ttt = int(input())
for _ in range(ttt):
    h, w, n = map(int, input().split(" "))

    num, floor = divmod(n, h)
    floor *= 100
    num += 1

    if floor == 0: print(h*100 + num -1)
    else: print(floor + num)

File: baekjoon/step/10250.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    int t, h, w, n;
    cin >> t;

    while(t--) {
        cin >> h >> w >> n;
        int floor = (n % h) * 100;
        int num = int(n/h) + 1;
        
        if (floor == 0) cout << h*100 + num -1 << '\n';
        else cout << floor + num << '\n';
    }
} 
